eZ/Publish/Core/Persistence/Legacy/Content/FieldValue/Converter/IntegerConverter.php

```diff
@@ -121,20 +121,20 @@ public function getIndexColumn()
      *   - {@link self::HAS_MAX_VALUE}
      *   - {@link self::HAS_MIN_VALUE}.
      *
-     * @param int|null $minValue Minimum int value, or null if not set
-     * @param int|null $maxValue Maximum int value, or null if not set
+     * @param int|false|null $minValue Minimum int value, false if none, or null if not set
+     * @param int|false|null $maxValue Maximum int value, false if none, or null if not set
```

Comment:
Sorry, I'm still confused :) You mentioned that the default value is `false`, so when is it `null`? When you edit the content type without giving this input it any value?